src/app/types/roles/CustomerRolesCollection.ts

```diff
@@ -0,0 +1,9 @@
+import { Collection, ObjectId } from 'mongodb';
+
+export interface CustomerRolesCollection extends Collection {
+	_id: ObjectId,
+    customerId: string,
```

Comment:
is it customerId or customer_id?

Reply:
I'll set both for now. Migrating to just `customerId` can be an onboarding bounty at some point